pkg/ingester/ingester.go

```diff
@@ -551,7 +551,7 @@ func (i *Ingester) Push(ctx context.Context, req *logproto.PushRequest) (*logpro
 	return &logproto.PushResponse{}, err
 }
 
-func (i *Ingester) GetOrCreateInstance(instanceID string) *instance {
+func (i *Ingester) GetOrCreateInstance(instanceID string) *instance { //nolint:revive
```

Comment:
This change seems unrelated.

Reply:
The new linter version reports: `unexported-return: exported method GetOrCreateInstance returns unexported type *ingester.instance`